server.go

```diff
@@ -469,6 +472,9 @@ func (s *Server) setupClient() error {
 		),
 	}
 
+	s.channelAcceptor = NewChannelAcceptor(s.lndServices.Client)
+	s.fundingManager.NotifyShimCreated = s.channelAcceptor.ShimRegistered
```

Comment:
set it directly when creating the struct?

Reply:
Done :+1: